mlflow/genai/evaluation/base.py

```diff
@@ -272,6 +274,7 @@ def predict_fn(question: str) -> str:
     return result
 
 
+@record_usage_event(GenAIEvaluateEvent)
```

Comment:
just to confirm, this won't lead to double count of events as evaluate_oss does not invoke `mlflow.models.evaluate`?